qml/SettingsView.qml

```diff
@@ -94,26 +94,21 @@ ScrollView {
     }
 
     function removePassword() {
-        yubiKey.validate(currentPasswordField.text, false, function (resp) {
-            if (resp.success) {
-
-                // TODO: Change to new method
-                yubiKey.setPassword(null, true, function (resp) {
-                    if (resp.success) {
-                        navigator.snackBar("Password removed")
-                        yubiKey.currentDeviceHasPassword = false
-                        passwordManagementPanel.isExpanded = false
-                    } else {
-                        navigator.snackBarError(getErrorMessage(resp.error_id))
-                        console.log(resp.error_id)
-                    }
-                    clearPasswordFields()
-                })
-            } else {
-                navigator.snackBarError(getErrorMessage(resp.error_id))
-                console.log(resp.error_id)
-            }
-        })
+        if (resp.success) {
```

Comment:
This change doesn't look right, `resp` is returned from `yubikey.py` , but it is being checked directly